virt/virt-4-9-release-notes.adoc

```diff
@@ -46,6 +46,9 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 
 //CNV-9540 High performance windows
 
+//CNV-8875
+* If your {VirtProductName} Operator subscription used any update channel other than *stable*, it is now automatically subscribed to the *stable* channel. The *stable* channel subscribes you to minor version updates and ensures that your {VirtProductName} version is compatible with your {product-title} version.
```

Comment:
Suggestion for the second sentence - "This single update channel delivers z-stream and minor version updates and ensures that your {VirtProductName} and {product-title} versions are compatible."

Reply:
I like the sound of that better. Will revise.